src/mono/mono/mini/mini-generic-sharing.c

```diff
@@ -2273,7 +2273,8 @@ instantiate_info (MonoMemoryManager *mem_manager, MonoRuntimeGenericContextInfoT
 		mono_class_setup_vtable (info->klass);
 		// FIXME: Check type load
 		if (mono_class_is_interface (iface_class)) {
-			ioffset = mono_class_interface_offset (info->klass, iface_class);
+			gboolean variance_used;
+			ioffset = mono_class_interface_offset_with_variance (info->klass, iface_class, &variance_used);
```

Comment:
Shouldn't we do something with variance_used ?

Reply:
For interface with generic variant arguments, it tries to find the compatible one and sets `variance_used = TRUE`. Result of the function is used for vtable method positioning as `method = m_class_get_vtable (info->klass) [ioffset + slot];`. I don't see that it should be changed according to `variance_used` variable. Other references of the method don't use `variance_used` as well.

Do you see a case where it should be used?